flux-pipe/relax_pfss_world.pdl

```diff
@@ -220,6 +220,9 @@ sub relax_pfss_world {
         # my $short_world_out_path = shorten_path($world_out_dir);
 
         # # print "\n\tWorld saved to $short_world_out_path";
+    # print $this_world_orig;
```

Comment:
Commented out lines as an addition here?

Reply:
Yeah, I guess I could switch it to "if False: print"? Or do you want all commented lines to go away?

Reply:
I'm going to stop commenting on commented out code, but I think it'd be good practice to tidy up old code that's no longer being used. Maybe review all of your changed files here to scan for them and remove any that aren't needed in one single commit?